README.md

```diff
@@ -23,3 +23,41 @@ make autorest
 [releases]: https://github.com/vshn/stardog-userrole-operator/releases
 [license]: https://github.com/vshn/stardog-userrole-operator/blob/master/LICENSE
 [dockerhub]: https://hub.docker.com/r/vshn/stardog-userrole-operator
+
+## Local Development
+
+### Prerequisites
+
+- minikube
+- helm
+- kubectl
+- Stardog image registry credentials
+- Stardog license
```

Comment:
Do we have any documentation (internal is fine) regarding where to find this information?

Reply:
Added a link to an internal wiki page